src/org/daisy/dotify/formatter/impl/obfl/ObflParserImpl.java

```diff
@@ -1052,6 +1052,10 @@ private BlockProperties blockBuilder(StartElement el) {
                 builder.textIndent(Integer.parseInt(att.getValue()));
             } else if ("first-line-indent".equals(name)) {
                 builder.firstLineIndent(Integer.parseInt(att.getValue()));
+            } else if ("right-text-indent".equals(name)) {
+                builder.rightTextIndent(Integer.parseInt(att.getValue()));
+            } else if ("right-last-line-indent".equals(name)) {
+                logger.warning("right-last-line-indent attribute is not supported, ignoring." + toLocation(el));
```

Comment:
1. Why? Is this so hard to implement?
2. This may result in an odd result when the last line is longer than the text width minus the indent. It seems to me that there is no way to have an right indent on the last line.
3. You may consider throwing an exception - so that there are no unexpected results

Reply:
Thanks for checking out the changes.
1. This is indeed not so straightforward to implement. And because I can't really think of a use case for `right-last-line-indent` at the moment, I preferred to not put time in it.
2. I don't understand, can you give an example?
3. I thought about that but wasn't sure. Maybe you're right.

Reply:
Concerning 2, this has to do with how `right-text-indent` in OBFL is defined. It reads "Right indent of every line in the block except the last one (in characters)". I already commented in the OBFL PR that I expected that you would have here "Right indent of every line in the block" and then with attribute `right-last-line-indent` you can override the indentation of the last line. That would seem to me a better approach, because why would you ever want to right-indent all lines BUT the last one? Your response was that this was because of symmetry with the current `text-indent` and `first-line-indent` attributes.

Here is the example. Suppose the page width is 8 and the right-text-indent is 3, then you can have this kind of behavior:

```
⠤⠤
⠤⠤
⠤⠤
⠤⠤⠤
```

As you can see, the last line of this block sticks out. That is kind of strange.

Reply:
😄 But this is the whole point of this feature. Check https://github.com/mtmse/obfl/pull/9.

Reply:
Oh? I didn't realize that this was meant intentionally